### PR TITLE
Add cors headers to components' server admin routes

### PR DESCRIPTION
**Pull request by bdoyle0182**
## Description
This pr adds cors headers to the server admin routes of each component plus the options route which is needed for pre-flight requests from browsers when auth is included in the request. The reason for this is to lay the ground work for future ui work for an admin portal.

## Related issue and scope
- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components
- [ ] API
- [X] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [X] Scheduler
- [X] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes
- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [X] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:
- [X] I signed an [Apache CLA](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md).
- [X] I reviewed the [style guides](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md#coding-standards) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

File: core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerPool.scala [hunk 1]
```diff
@@ -78,6 +76,7 @@ case class DeletionContainer(deletionMessage: ContainerDeletionMessage)
 case object Remove
 case class Keep(timeout: FiniteDuration)
 case class PrewarmContainer(maxConcurrent: Int)
+case object GetState
```
Comment (style95) [inline core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerPool.scala:79]: Is there any reason we should define another `GetState` here?

File: core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala [hunk 2]
```diff
@@ -152,7 +152,6 @@ case class InitializedData(container: Container,
                            override val clientProxy: ActorRef)
     extends ContainerAvailableData(container, invocationNamespace, action)
     with WithClient {
-  override def getContainer = Some(container)
```
Comment (style95) [inline core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala:155]: Why are these removed?

**Review by style95 — COMMENTED** [review]
It generally looks good to me.
BTW, do you have the plan to contribute an admin UI too?

**Comment by bdoyle0182** [comment]
> 

Not at the moment don't have bandwidth to do any ui work, but maybe down the road

File: core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerPool.scala (hunk 1 above)
Comment (bdoyle0182) [reply core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerPool.scala]: I see what happened now, I had this in progress for awhile and had a bad merge somewhere with your recent commits. Should all be cleaned up now.

File: core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala (hunk 2 above)
Comment (bdoyle0182) [reply core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala]: fixed

**Review by style95 — APPROVED** [review]
